### PR TITLE
WIP/incomplete - add attempt for search functionality

### PR DESCRIPTION
**Pull request by trblackw**
@alexweininger I'm not sure of the github handle of who I was talking to on discord (mac #4164), but we discussed where I was at with the search functionality and I told him I'd push what I had so far.

I've tried a variety of ways of implementing search functionality but none have seemed to correctly update the files shown. In this PR, I did modify the way in which filenames are displayed. 

NOTE: this is *not* ready to be merged, I just wanted to share where I was at so far and see if anyone had some thoughts on a working direction to take.

### CODE REVIEW

**Comment by marcosnils** [comment]
@trblackw seems it wasn't me?

**Comment by alexweininger** [comment]
It was me :)

File: src/options.ts [hunk 1]
```diff
@@ -1,35 +1,33 @@
...
 // stored in chrome.storage.
-function restore_options() {
-    // Use default value color = 'red' and likesColor = true.
-    chrome.storage.sync.get(
-        {
-            defaultTab: "local"
-        },
-        function(items) {}
-    );
+function restore_options(): void {
```
Comment (alexweininger) [inline src/options.ts]: Great changes.

File: src/options.ts [hunk 2]
```diff
@@ -1,35 +1,33 @@
 /// <reference path='../node_modules/@types/chrome/index.d.ts'/>
-"../node_modules/@types/chrome/index.d.ts";
+'../node_modules/@types/chrome/index.d.ts';
 // Saves options to chrome.storage
-function save_options(): any {
-	var defaultTab: HTMLCollectionOf<HTMLOptionElement> = (<HTMLSelectElement> document.getElementById("default-tab")).selectedOptions;
-	console.log(defaultTab[0].value);
+function save_options(): void {
+   var defaultTab: HTMLCollectionOf<HTMLOptionElement> = (<HTMLSelectElement>document.getElementById('default-tab')).selectedOptions;
```
Comment (alexweininger) [inline src/options.ts]: I would like to decide on a style to enforce across the entire repo. For now pretty much anything goes. Do you have any experience with linters?

**Review by alexweininger — CHANGES_REQUESTED** [review]
Just adding this so that there is no chance this gets merged.

File: src/options.ts (hunk 2 above)
Comment (trblackw) [reply src/options.ts]: Yes actually. I'll add one, push it to this PR and tell me what you think.

Comment (alexweininger) [reply src/options.ts]: Ok - could it be enforced as a code check on PRs?

File: .prettierrc [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
+  "trailingComma": "es5",
+  "tabWidth": 3,
```
Comment (alexweininger) [inline .prettierrc]: 4 please :)

**Review by alexweininger — CHANGES_REQUESTED** [review]

**Pull request closed without merging**